src/Symfony/Bundle/SecurityBundle/Tests/DependencyInjection/Compiler/AddSecurityVotersPassTest.php

```diff
@@ -73,9 +73,9 @@ public function testThatSecurityVotersAreProcessedInPriorityOrder()
     }
 
     /**
-     * Test that in debug mode, voters are correctly decorated.
+     * Test that in debug mode, voters are correctly traceable
```

Comment:
dot should not be removed, but I would remove the docblock here instead.

Reply:
done.